src/main/java/org/skriptlang/skript/lang/structure/Structure.java

```diff
@@ -41,9 +41,9 @@
 import org.skriptlang.skript.lang.entry.EntryContainer;
 import org.skriptlang.skript.lang.entry.EntryData;
 import org.skriptlang.skript.lang.entry.EntryValidator;
+import org.skriptlang.skript.lang.script.Annotation;
 
-import java.util.Arrays;
-import java.util.Iterator;
+import java.util.*;
```

Comment:
I don't think a star import is necessary (for what appears to be) < 5 java.util classes